src/portal/src/app/config/gc/gc.api.repository.ts

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
I noticed the email and auth all contains email and auth in file name. I think this gc should be keeped., Because it is easier when you find this file from vscode. Without gc, there maybe many files with same name. It will increase difficulties.